setup.py

```diff
@@ -27,4 +27,11 @@
         'wfa_cardinality_estimation_evaluation_framework.evaluations',
         'wfa_cardinality_estimation_evaluation_framework.evaluations.data',
         'wfa_cardinality_estimation_evaluation_framework.simulations',
-    ])
+    ],
+    entry_points={
+        'console_scripts': [
+            'run-evaluation=wfa_cardinality_estimation_evaluation_framework'
```

Comment:
I wonder if the name of the global script should be a bit more specific like wfa-run-evaluation so it's even more clear the console script comes from this repo, and tab completion can work off of wfa instead of run?

Reply:
Resolved in 623deb4